deepmd/main.py

```diff
@@ -651,6 +733,16 @@ def main():
         if no command was input
     """
     args = parse_args()
-    from deepmd.tf.entrypoints.main import main as deepmd_main
+
+    if args.tf:
```

Comment:
we may want to check `--tf`` and `--pt` should be be set at the same time.

Reply:
This is what `add_mutually_exclusive_group` has done

Reply:
It seems there is a better code here:
https://stackoverflow.com/a/62863301/9567349